.github/workflows/labels.yml

```
@@ -191,6 +191,11 @@ jobs:
                     Closing this issue. If you believe this issue has been closed in error, please provide any relevant output and logs which may be useful in diagnosing the issue.
                   `
                 },
+                attic: {
+                  close: true,
+                  comment: `
+                    Thanks for your contribution to Metasploit Framework! We've looked at this issue, and we agree that it seems like a good issue to resolve - however we do not currently have the bandwidth to prioritize this issue. We've labeled this as \`attic\` and closed it for now. If you believe this issue has been closed in error, or that it should be prioritized, please comment with additional information.`
+                }
```

To be consistent with the other bot responses:

```suggestion
                  comment: `
                    Thanks for your contribution to Metasploit Framework! We've looked at this issue, and we agree that it seems like a good issue to resolve - however we do not currently have the bandwidth to prioritize this issue.
                    
                    We've labeled this as \`attic\` and closed it for now. If you believe this issue has been closed in error, or that it should be prioritized, please comment with additional information.
                    `
                }
```